README.md

```diff
@@ -46,6 +48,72 @@ echo "\n\nAttachments:\n";
 print_r($mail->getAttachments());
 ```
 
+Example 2:
```

Comment:
There is no need to copy/paste connection for every single example, you can just add different cases lines in basic example with related comments.